doc_source/troubleshooting-ch.md

```diff
@@ -18,15 +18,13 @@ The following information might help you troubleshoot common issues when using t
 
  The default version of Git released on OS X and macOS uses the Keychain Access utility to save generated credentials\. For security reasons, the password generated for access to your AWS CodeCommit repository is temporary, so the credentials stored in the keychain will stop working after about 15 minutes\. If you are only accessing Git with AWS CodeCommit, try the following:
 
-1. Using Terminal, determine where Git is installed on the local machine:
+1. Find your Git configuration file\. You can use the git option `--show-origin` to find out the config file where Mac OS X Keychain credential helper is defined:
 
    ```
-   $ which git
-   
-   /usr/local/git/bin/git
+   $ git config -l —show-origin
```

Comment:
Thank you for this contribution! From my testing, I believe this must have two dashes before show-origin, not one?  
`git config -l --show-origin`

Reply:
Yes, my bad, editor ate the second :) Updating PR